SAP/luigi
# Clean Luigi store

We do not clear the subscriptions that we store in Luigi store. That leads to multiple initialization of route handling.

How to reproduce:
Browser console:
```
var cfg = Luigi.getConfig();
Luigi.unload();
Luigi.setConfig(cfg);
Luigi.unload();
Luigi.setConfig(cfg);
```


Result: 
=> navigate: luigi-client did not respond, using fallback by replacing iframe

Root cause:
How to check the multiple initialization of route handling:
1. add a console log [here](https://github.com/SAP/luigi/blob/main/core/src/App.svelte#L366) console.log('enableRouting')
2. start simpledev
3. browser console 
```
var cfg = Luigi.getConfig();
Luigi.unload();
Luigi.setConfig(cfg);
Luigi.unload();
Luigi.setConfig(cfg);

// when you do this more often you will see an increase of the console logs 'enableRouting'
Luigi.unload()
Luigi.setConfig(cfg);
Luigi.unload()
Luigi.setConfig(cfg);
....
```
